### PR TITLE
Added new tableExport library and export ability on shortlist page

### LINKED ISSUE #54: Ability to export shortlist to excel
Would be great to have the ability to export your shortlisted jobs to an external document (table) such as Excel

### PR DESCRIPTION
**Pull request by Neil511**
Fixes #54

### CODE REVIEW

**Comment by Neil511** [comment]
Will minify 3rd party JS files.

**Comment by gurpreetg** [comment]
@Neil511 please only include the required files (jquery and actual code) as we talked in person!

File: manifest.json [hunk 1]
```diff
@@ -28,6 +28,8 @@
         "js/external/jquery.min.js",
         "js/external/simple-modal.js",
         "js/external/mustache.min.js",
+        "js/external/tableExport/tableExport.js",
+        "js/external/tableExport/jquery.base64.js",
```
Comment (peygar) [inline manifest.json]: move this next to the other jquery

File: js/external/tableExport/tableExport.js [hunk 2]
```diff
@@ -0,0 +1,359 @@
+/*The MIT License (MIT)
```
Comment (peygar) [inline js/external/tableExport/tableExport.js]: no minify?

File: js/external/tableExport/jquery.base64.js [hunk 3]
```diff
@@ -0,0 +1,190 @@
+/*jslint adsafe: false, bitwise: true, browser: true, cap: false, css: false,
```
Comment (peygar) [inline js/external/tableExport/jquery.base64.js]: minify?

File: js/content_script.js [hunk 4]
```diff
@@ -547,14 +547,35 @@ function insertShowHideNewButton() {
...
 
+function insertExportAsExcelButton() {
+	var imgURL = "url(" + chrome.extension.getURL("assets/gooseButtonIcon.png") + ")";
+	$("<button \ \
+		class='goose-button'\
+		type='button'>Export Table\
+	</button>")
+	.css('background-image', imgURL)
+	.click(exportTableToExcel)
+	.appendTo($(".tab-content .row-fluid:eq(0) .span12 .aaaa .row-fluid:eq(0)"));
```
Comment (peygar) [inline js/content_script.js]: lol `.aaaa`

**Comment by brijendra98** [comment]
Could you add the tableExport feature to all pages (applied, viewed,etc.) and not just the shortlist?

**Comment by peygar** [comment]
Yeah we should be able to do that soon

**Pull request merged**